tap_dashboard_reports/streams.py

```diff
@@ -31,10 +33,13 @@ def name(self):
 
     @property
     def primary_keys(self):
```

Comment:
keys = self._dimensions
if self.custom_key:
     keys.append(self.custom_key)

return keys

Reply:
change def primary_keys for all scenarios:
- only custom_keys
- only group by columns
- both custom_keys and  group by columns